File: atsa/src/main/java/jp/kaiz/atsassistmod/client/ClientNetworkHelper.java
// SPDX-License-Identifier: LGPL-3.0-or-later
// Copyright © 2026 mirukuneko and RealTrainModRenewed contributors
package jp.kaiz.atsassistmod.client;

import net.minecraft.client.Minecraft;
import net.minecraft.network.protocol.common.custom.CustomPacketPayload;

public final class ClientNetworkHelper {
    private ClientNetworkHelper() {
    }

    public static void sendToServer(CustomPacketPayload payload) {
        if (Minecraft.getInstance().getConnection() != null) {
            Minecraft.getInstance().getConnection().send(payload);
        }
    }
}

File: atsa/src/main/kotlin/jp/kaiz/atsassistmod/client/ClientNetworkHelper.kt
// SPDX-License-Identifier: LGPL-3.0-or-later
// Copyright © 2026 mirukuneko and RealTrainModRenewed contributors
package jp.kaiz.atsassistmod.client

import net.minecraft.client.Minecraft
import net.minecraft.network.protocol.common.custom.CustomPacketPayload

object ClientNetworkHelper {
    @JvmStatic
    fun sendToServer(payload: CustomPacketPayload) {
        Minecraft.getInstance().connection?.send(payload)
    }
}
